File: Objective-C/Neocom/Neocom/UIViewController+NC.m
//
//  UIViewController+NC.m
//  Neocom
//
//  Created by Artem Shimanski on 18.11.16.
//  Copyright © 2016 Artem Shimanski. All rights reserved.
//

#import "UIViewController+NC.h"

@implementation UIViewController (NC)

- (__kindof UIViewController*) topmostViewController {
	UIViewController* controller;
	for (controller = self; controller.presentedViewController; controller = controller.presentedViewController);
	return controller;
}

@end

File: Neocom/Neocom/UIViewController+NC.swift
//
//  UIViewController+NC.swift
//  Neocom
//
//  Created by Artem Shimanski on 01.02.17.
//  Copyright © 2017 Artem Shimanski. All rights reserved.
//

import UIKit

extension UIViewController {
	@IBAction func dismissAnimated(_ sender: Any) {
		dismiss(animated: true, completion: nil)
	}
}
